Cargo.toml

```diff
@@ -228,16 +228,18 @@ tokio-stream = "0.1.16"
 # We *always* pin the latest version of protocol to disallow accidental changes in the execution logic.
 # However, for the historical version of protocol crates, we have lax requirements. Otherwise,
 # Bumping a crypto dependency like `boojum` would require us to republish all the historical packages.
-circuit_encodings = { package = "circuit_encodings", version = "=0.150.18" }
-circuit_sequencer_api = { package = "circuit_sequencer_api", version = "=0.150.18" }
-crypto_codegen = { package = "zksync_solidity_vk_codegen", version = "=0.30.11" }
-kzg = { package = "zksync_kzg", version = "=0.150.18" }
+circuit_encodings = "0.150.19"
+circuit_sequencer_api = "0.150.19"
+circuit_definitions = "0.150.19"
```

Comment:
It is a bad idea to unlock the version. The latest version of crypto/protocol crates _must_ be pinned to the last one; otherwise `cargo update` can bump the version in lockfile automatically.

Pls return `=` to all the crates where it was removed.

Reply:
cc @EmilLuta